SpectraLogic.EscapePodClient.Integration.Test/EscapePodClientIntegrationTests.cs

```diff
@@ -19,69 +19,77 @@
 using System;
 using System.Collections.Generic;
+using System.IO;
 using System.Threading;
 
 namespace SpectraLogic.EscapePodClient.Integration.Test
 {
     [TestFixture]
     public class EscapePodClientIntegrationTests
     {
-        #region Fields
+        #region Private Fields
 
         private ILog _log = LogManager.GetLogger("EscapePodClientIntegrationTest");
 
-        #endregion Fields
+        #endregion Private Fields
 
-        #region Tests
+        #region Public Methods
 
         [Test]
         public void ArchiveAndRestore()
         {
-            /***********
-             * ARCHIVE *
-             ***********/
-            var fileName1 = Guid.NewGuid().ToString();
-            var fileName2 = Guid.NewGuid().ToString();
-            var archiveRequest = new ArchiveRequest(EscapePodClientFixture.ArchiveName, new List<ArchiveFile>
+            try
             {
-                new ArchiveFile(fileName1, "file:///C:/Users/sharons/Documents/GitHub/ep_net_sdk/SpectraLogic.EscapePodClient.Integration.Test/TestFiles/F1.txt", 14, new Dictionary<string, string>{ { "fileName", fileName1 } }, false, false),
-                new ArchiveFile(fileName2, "file:///C:/Users/sharons/Documents/GitHub/ep_net_sdk/SpectraLogic.EscapePodClient.Integration.Test/TestFiles/F2.txt", 14, new Dictionary<string, string>{ { "fileName", fileName2 } }, false, false)
-            });
+                /***********
+                 * ARCHIVE *
+                 ***********/
+                var fileName1 = Guid.NewGuid().ToString();
+                var fileName2 = Guid.NewGuid().ToString();
+                var archiveRequest = new ArchiveRequest(EscapePodClientFixture.ArchiveName, new List<ArchiveFile>
+                {
+                    new ArchiveFile(fileName1, $"{EscapePodClientFixture.ArchiveTempDir}/F1.txt".ToFileUri(), 14, new Dictionary<string, string>{ { "fileName", fileName1 } }, false, false),
+                    new ArchiveFile(fileName2, $"{EscapePodClientFixture.ArchiveTempDir}/F2.txt".ToFileUri(), 14, new Dictionary<string, string>{ { "fileName", fileName2 } }, false, false)
+                });
 
-            var archiveJob = EscapePodClientFixture.EscapePodClient.Archive(archiveRequest);
+                var archiveJob = EscapePodClientFixture.EscapePodClient.Archive(archiveRequest);
 
-            do
-            {
-                archiveJob = EscapePodClientFixture.EscapePodClient.GetJob(new GetEscapePodJobRequest(EscapePodClientFixture.ArchiveName, archiveJob.JobId));
-                _log.Debug(archiveJob.Status);
-                Thread.Sleep(5000);
-            } while (archiveJob.Status.Status == JobStatus.ACTIVE);
+                do
+                {
+                    archiveJob = EscapePodClientFixture.EscapePodClient.GetJob(new GetEscapePodJobRequest(EscapePodClientFixture.ArchiveName, archiveJob.JobId));
+                    _log.Debug(archiveJob.Status);
+                    Thread.Sleep(5000);
```

Comment:
You should probably add a max number of attempts so that your code doesn't go into an infinite loop while I'm still working on the error reporting.